src/CoffeeMatch.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const CoffeeMatch = () => {
+  [
+    {
+      name,
+      image,
+      qr,
+    },
+  ];
+  return (
+    <div>
+      <h3>CoffeeMatch</h3>
+      <div>
+        {items.map((item) => (
```

Comment:
Let's iterate through the `matches` prop here:

```suggestion
        {matches.map((item) => (
```